subsys/bootloader/bl_storage/bl_storage.c

```diff
@@ -332,3 +349,81 @@ int set_monotonic_counter(uint16_t new_counter)
 	write_halfword(next_counter_addr, ~new_counter);
 	return 0;
 }
+
+/* OTP is 0xFFFF after erase so setting all bits to 0 so a full erase is needed
+ * to reset
+ */
+#define STATE_LEFT 0x0000
+
+int write_life_cycle_state(lcs_t next_lcs)
```

Comment:
the function names, structs etc. added here should probably be prefixed with bl_ ?

Reply:
Well the rest of the functions in the file don't either so that's with just "copied" what was there already

Reply:
I guess this is fine then :)